pkg/types/label.go

```diff
@@ -20,14 +20,15 @@ const (
 	ContainerLabelRStudioServerAddr = "ai.tensorchord.envd.rstudio.server.address"
 	ContainerLabelSSHPort           = "ai.tensorchord.envd.ssh.port"
 
-	ImageLabelVendor  = "ai.tensorchord.envd.vendor"
-	ImageLabelGPU     = "ai.tensorchord.envd.gpu"
-	ImageLabelAPT     = "ai.tensorchord.envd.apt.packages"
-	ImageLabelPyPI    = "ai.tensorchord.envd.pypi.packages"
-	ImageLabelR       = "ai.tensorchord.envd.r.packages"
-	ImageLabelCUDA    = "ai.tensorchord.envd.gpu.cuda"
-	ImageLabelCUDNN   = "ai.tensorchord.envd.gpu.cudnn"
-	ImageLabelContext = "ai.tensorchord.envd.build.context"
+	ImageLabelVendor    = "ai.tensorchord.envd.vendor"
+	ImageLabelGPU       = "ai.tensorchord.envd.gpu"
+	ImageLabelAPT       = "ai.tensorchord.envd.apt.packages"
+	ImageLabelPyPI      = "ai.tensorchord.envd.pypi.packages"
+	ImageLabelR         = "ai.tensorchord.envd.r.packages"
+	ImageLabelCUDA      = "ai.tensorchord.envd.gpu.cuda"
+	ImageLabelCUDNN     = "ai.tensorchord.envd.gpu.cudnn"
+	ImageLabelContext   = "ai.tensorchord.envd.build.context"
+	ImageLabelCacheHash = "ai.tensorchord.envd.build.manifestBytecodeHash"
```

Comment:
Should we use ai.tensorchord.envd.build.hash?

Reply:
I think build.hash is no a good idea. But I really need a short term. The hash digested from the bytecode of `build.envd`. 
@VoVAllen @kemingy

Reply:
How about `manifestHash`? What does byte code mean here?

Reply:
It is the hash of manifest's bytecode not manifest directly.

Reply:
* short one: `ai.tensorchord.envd.build.digest`
* including the implementation: `ai.tensorchord.envd.build.bytecodeHash`

Reply:
the first LGTM, thx!